driver-core/src/main/com/mongodb/internal/operation/CommandCursorResult.java

```diff
@@ -84,11 +94,21 @@ public List<T> getResults() {
      *
      * @return the server address
      */
-    public ServerAddress getAddress() {
+    public ServerAddress getServerAddress() {
         return serverAddress;
     }
 
     public long getCursorId() {
         return cursorId;
     }
+
```

Comment:
These methods are unused, even by tests?  Maybe they are used in a subsequent PR?

Reply:
Correct - these PRs are just extra reworkings of the original PR.